lib/routes/devices-sessions.js

```diff
@@ -256,7 +265,8 @@ module.exports = (log, db, config, customs, push, devices) => {
             type: DEVICES_SCHEMA.type.required(),
             pushCallback: DEVICES_SCHEMA.pushCallback.allow(null).optional(),
             pushPublicKey: DEVICES_SCHEMA.pushPublicKey.allow(null).optional(),
-            pushAuthKey: DEVICES_SCHEMA.pushAuthKey.allow(null).optional()
+            pushAuthKey: DEVICES_SCHEMA.pushAuthKey.allow(null).optional(),
+            pushEndpointExpired: DEVICES_SCHEMA.pushEndpointExpired.required()
```

Comment:
nit: should this be `required` even is non of the other push-related items are present?